### DISCUSSION TITLE
Proes not working correctly on mobile browsers (android)

### DISCUSSION THREAD

**Discussion by mujehoxe** [tailwindlabs/tailwindcss#9886] [Help]
I have these two components 

PageHeader.vue
 ```
 <header class="px-4 pt-6 prose-sm md:px-6 md:prose">
    <h1>{{ props.pageTitle }}</h1>
  </header>
```

CourseForm.vue
```
<template>
  <PageHeader page-title="Course Creation Form"/>
  <main>
     <div class="p-4 mx-auto prose md:px-6 prose-indigo sm:rounded-md">
        <form class="space-y-8 divide-y divide-gray-200">
        <div class="space-y-8 divide-y divide-gray-200 sm:space-y-5">

       <div>
         <div class="col-span-full">
            <h3 class="text-lg leading-6 font-medium text-gray-900">Course Information</h3>
  ...
```
On desktop browser device i get
![](https://cdn.discordapp.com/attachments/1042412217632555019/1042412217749999657/image.png)

And i get this on an actual android mobile browser
![](https://cdn.discordapp.com/attachments/1042412217632555019/1042412218056196127/image.png)

When i removed prose-sm from <header> the rendering on the desktop browser mobile browser became identical, which makes me think that mobile isn't accepting the prose-sm class.

Tried it also on a genymotion vm
![](https://cdn.discordapp.com/attachments/1042412217632555019/1042436734736150558/image.png)

Also tried providing the proes and proes-base classes like so : `<header class="px-4 pt-6 prose prose-sm md:px-6 md:prose-base">` but didn't notice any effect.

**Comment by wongjn** [comment]
How do you expect the page to look on the mobile browsers? Could it be that that the desktop browser matches the `md:` breakpoint but the mobile viewports do not (or vice-versa)? Since you are attempting to use `prose-sm` for smaller viewports, this would make the text smaller, depending on how the `@tailwind/typography` styles are configured. If you make the desktop viewport smaller such that it does not match the `md:` breakpoint (if it did not already), how does it look?

**Reply by wongjn** [reply]
> Both views are under the same breakpoints since the desktop browser is trying to mimic a mobile device, the look i expect is that the "create course form" should have more emphasis than "Course Information" and to achieve that i currently am using the font-bold and text-2xl which is ugly both in rendering and in code.[^1]

Thank you for that additional information! Do you have a remotely-hosted git repository we could take a closer at to try to ascertain the problem please?

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/9886#discussioncomment-4190234

**Reply by mujehoxe** [reply]
I guess you are the same person i talked to about this issue on discord discord.com/channels/486935104384532500/1042412217632555019
and i already provided the link to my remote repo

**Reply by wongjn** [reply]
This is how it looks for me on desktop in Chrome:
![image](https://user-images.githubusercontent.com/11310624/202998769-4f84496b-096c-4f85-9c10-dfa8a289f475.png)

And on an Android device, in Chrome:
![image](https://user-images.githubusercontent.com/11310624/202999679-5c67f953-a34c-4133-af3b-76f30b1806e8.png)

I believe this is the behavior you'd like? Perhaps try [remote debugging](https://developer.chrome.com/docs/devtools/remote-debugging/) on the your Android device to try to get more clues about why it does not work as intended for you. Perhaps also check you are serving up-to-date compiled CSS.

**Reply by mujehoxe** [reply]
Well certainly they look different. Are you making sure to enabled the "Toggle device toolbar" option?
![image](https://user-images.githubusercontent.com/52047683/203106564-aa7548d6-045b-4557-bf3e-c8755014aeb6.png)

Yes i tried remote debugging

**Reply by wongjn** [reply]
The desktop Chrome screenshot was taken with the _Toggle device toolbar_ enabled.

**Comment by mujehoxe** [comment]
Both views are under the same breakpoints since the desktop browser is trying to mimic a mobile device, the look i expect is that the "create course form" should have more emphasis than "Course Information" and to achieve that i currently am using the font-bold and text-2xl which is ugly both in rendering and in code.

**Reply by wongjn** [reply]
Thank you for the information! I've responded here https://github.com/tailwindlabs/tailwindcss/discussions/9886#discussioncomment-4190245 to better keep the reply chain together.
